script/Ciel/Day 7/07_01_CIEL07_5.txt

```diff
@@ -923,7 +923,7 @@ She's counting on me...?
 // temptress that seduces men seems a bit redundant, maybe truncate to just 'seductress'?
 // まさにその通りだ。 could be translated in a million ways, so be creative.
 I feel like there was another meaning to the word vampire...
- Maybe that a 'vamp' can also refer to a temptress that seduces men.
+ Maybe a 'vamp' can also refer to a temptress that seduces men.
```

Comment:
Gotta keep the `that` here imo - otherwise it sounds like he's hypothesizing rather than stating what is in fact a real definition.